src/aspire/commands/extract_particles.py

```diff
@@ -37,6 +38,27 @@
     is_flag=True,
     help="Set this flag if coordinate files contain (X,Y) particle centers",
 )
+@click.option(
+    "--downsample",
+    default=0,
+    type=int,
+    help="Downsample the data to this resolution prior to saving to starfile/.mrcs stack",
+)
+@click.option(
+    "--normalize_bg",
+    is_flag=True,
+    help="Normalize images to the background noise",
+)
+@click.option(
+    "--whiten",
+    is_flag=True,
+    help="Estimate the noise variance of the images and whiten",
+)
+@click.option(
+    "--invert_contrast",
+    is_flag=True,
+    help="Invert the contrast of the images so molecules are shown in white",
```

Comment:
White or black depends on the colormap. Maybe just say that the contrast is inverted so that the clean particles are positive. Also this is only done if the intensity is estimated to be negative.